### PR TITLE
Updated sync-mainframe-data-with-azure-content.md  with Azure Synapse…

### PR DESCRIPTION
**Pull request by msdnguy**
… integration option

Azure Synapse native DB2 connector allows direct integration to Mainframe and Midrange platforms.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@EdPrice-MSFT: I'm sorry - only the author of this article, @, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by bhbandam — APPROVED** [review]

**Review by bhbandam — APPROVED** [review]

**Comment by bhbandam** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@bhbandam: I'm sorry - only the author of this article, @, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by bhbandam** [comment]
Can you please merge this request ?

Bhaskar Bandam
Senior Program Manager
Azure Global Engineering
Office: +1 703 673 7887 | Mobile: +1 703 581 3678 | Email: ***@***.******@***.***>
***@***.***

**Review by bhbandam — DISMISSED** [review]

File: docs/reference-architectures/migration/sync-mainframe-data-with-azure-content.md [hunk 1]
```diff
@@ -230,3 +232,4 @@ Use the [Azure pricing calculator][Azure pricing calculator] to estimate the cos
 [Azure Database for MySQL]: https://azure.microsoft.com/services/mysql/
 [What is HIS]: /host-integration-server/what-is-his
 [What is NoSQL? Databases for a cloud-scale future]: https://www.infoworld.com/article/3240644/what-is-nosql-databases-for-a-cloud-scale-future.html
+[Azure Synapse Analytics] /azure/synapse-analytics/overview-what-is
```
Comment (ckittel) [inline docs/reference-architectures/migration/sync-mainframe-data-with-azure-content.md:235]:
```suggestion
[Azure Synapse Analytics]: /azure/synapse-analytics/overview-what-is
```

File: docs/reference-architectures/migration/sync-mainframe-data-with-azure-content.md [hunk 2]
```diff
@@ -95,9 +95,11 @@ The solution uses the following components:
 
 #### Data integrators
 
-- [Data Factory][Azure Data Factory] is a hybrid data integration service. You can use this fully managed, serverless solution to create, schedule, and orchestrate ETL and [ELT][ELT] workflows.
+- [Azure Data Factory][Azure Data Factory] is a hybrid data integration service. You can use this fully managed, serverless solution to create, schedule, and orchestrate ETL and [ELT][ELT] workflows.
 
-- [SSIS][SQL Server Integration Services] is a platform for building enterprise-level data integration and transformation solutions. You can use SSIS to manage, replicate, cleanse, and mine data.
+- [Azure Synapse Analytics] is an enterprise analytics service that accelerates time to insight, across data warehouses and big data systems. Azure Synapse brings together the best of SQL technologies (that are used in enterprise data warehousing), Spark technologies used for big data, Data Explorer for log and time series analytics, Pipelines for data integration and ETL/ELT, and deep integration with other Azure services, such as Power BI, Cosmos DB, and Azure Machine Learning.
```
Comment (ckittel) [inline docs/reference-architectures/migration/sync-mainframe-data-with-azure-content.md:100]:
```suggestion
- [Azure Synapse Analytics][Azure Synapse Analytics] is an enterprise analytics service that accelerates time to insight, across data warehouses and big data systems. Azure Synapse brings together the best of SQL technologies (that are used in enterprise data warehousing), Spark technologies used for big data, Data Explorer for log and time series analytics, Pipelines for data integration and ETL/ELT, and deep integration with other Azure services, such as Power BI, Cosmos DB, and Azure Machine Learning.
```

**Review by ckittel — CHANGES_REQUESTED** [review]

**Review by ckittel — APPROVED** [review]
Lgtm

**Comment by msdnguy** [comment]
Hi Chad,

I approved the change.  Sorry about the sloppy markdown.

marc

**Pull request merged**